README.md

```diff
@@ -191,5 +191,16 @@ Tune-sklearn also supports the use of other machine learning libraries such as P
 * [Pytorch (Skorch)](https://github.com/ray-project/tune-sklearn/blob/master/examples/torch_nn.py)
 * [XGBoost](https://github.com/ray-project/tune-sklearn/blob/master/examples/xgbclassifier.py)
 
+## [Documentation](docs)
```

Comment:
Is this expected to have a link to the docs?

Reply:
Yeah, it's a working link to the docs overview page